core/src/main/scala/org/http4s/headers/Age.scala

```diff
@@ -7,20 +7,30 @@ import org.http4s.parser.HttpHeaderParser
 
 import scala.concurrent.duration.FiniteDuration
 import scala.concurrent.duration._
+import scala.util.Try
 
 object Age extends HeaderKey.Internal[Age] with HeaderKey.Singleton {
-  def apply(age: FiniteDuration): ParseResult[Age] =
-    if (age >= 0.seconds) {
+  def apply(age: Long): ParseResult[Age] =
+    if (age >= 0) {
       ParseResult.success(new Age(age) {})
     } else {
       ParseResult.fail("Invalid age value", s"Age param $age must be more or equal to 0 seconds")
     }
 
   def unsafeFromDuration(age: FiniteDuration): Age =
+    apply(age.toSeconds).fold(throw _, identity)
+
+  def unsafeFromLong(age: Long): Age =
     apply(age).fold(throw _, identity)
 
   override def parse(s: String): ParseResult[Age] =
     HttpHeaderParser.AGE(s)
+
+  // Render a finite duration in seconds
+  implicit val longRenderer: Renderer[Long] = new Renderer[Long] {
```

Comment:
One last thing: couldn't this go into renderer?  This has nothing to do with age.